requirements.txt

```diff
@@ -2,7 +2,7 @@ numpy
 accelerate>=0.20.3
 transformers>=4.34.1
 torch
-aim==3.17.5
+aim==3.18.1
```

Comment:
since there's a desire to implement multiple trackers, did we want to make the dependency (and imports) optional, just used when available and configured?

Reply:
can be done..but then do we still list these in the requirements.txt?

or do we throw an error and ask user to install the required tracker before importing it in the code.